configs/katello/4.1.yaml

```diff
@@ -70,6 +70,25 @@
   - 31618 # docs
   - 32165 # docs
   - 31880 # squashed, already in release branch
+  - 32334 # empty cherry-pick
```

Comment:
One of our tool belt environments is very wrong!

My list of cherry picks for 4.1 looks like this:

```
                                                                                                                                                                                                                                                                                
32459 - 2021-05-06 19:01:05 UTC: [5d9a25da057ce8cd6181995824f20129e233d960] Removal applicabilty code including event queue event, profile uploadings, applicability actions, config                                                                                            
29898 - 2021-05-14 20:01:17 UTC: [b033c5fae831e4233dd0a107841d4064fa0cc8f0] applicability should run once for a sync plan instead of after every repo                                                                                                                           31411 - 2021-05-14 20:01:22 UTC: [9e06c62b37c568f03466a155f3a801b101e25f21] Actions::Katello::Applicability::Hosts::BulkGenerate called many times with only a single host after sync plan runs                                                                                 32578 - 2021-05-17 17:01:17 UTC: [fa770a9ea3a14957e1c50425308e1a3fbf4244d6] Changing the HTTP proxy for a repository that doesn't have an upstream URL causes an error (nightly)                                                                                                
32591 - 2021-05-17 20:16:08 UTC: [7c9aa78fb5c5c14ea15af759a9d36fd4a7368377] katello-agent uses incorrect ping mechanism                                                                                                                                                         
32436 - 2021-05-19 16:01:19 UTC: [02e2151a90717faf54cb6c3918d6d1dc2f777ae5]  Errata installation via Host Collection Remote Execution Only Sends First Page of Errata Ids                                                                                                       
32447 - 2021-05-20 04:01:07 UTC: [be9468721324ae1962fb20eae763c65b2a519444] Katello agent tasks that error prevent re-running due to locked tasks                                                                                                                               
31996 - 2021-05-20 14:01:22 UTC: [1984006ea57a497a7619b5c0c1f61b32b52e86be] Remove deprecated API params                                                                                                                                                                        
32623 - 2021-05-20 15:01:10 UTC: [63a1d5bae152290cc0a31e9f1eea491749f8a009] katello:delete_orphaned_content fails on Katello 3.18 with "backend service pulp3: 503 Service Unavailable"                                                                                         
32533 - 2021-05-20 21:01:08 UTC: [8e5d21a269bcc236baf583e54677d05857746a70]  Duplicate YumMetadata index entries in content view repositories causing unneeded capsule sync                                                                                                     
32590 - 2021-05-26 16:01:14 UTC: [49b68610e1b676c4a2174422b45828a6f338d48d] 20210119162528_delete_puppet_and_ostree_repos.rb fails to remove puppet repos in CVs, leading to upgrade errors                                                                                     
32380 - 2021-05-26 18:01:08 UTC: [83a1270132a3f26ec4db98625e413b9d7068f01e] Error performing CreatePulpDiskSpaceNotifications  NameError (undefined local variable or method `percentage' for Katello::UINotifications::Pulp::ProxyDiskSpace:Class)                             
32137 - 2021-05-27 17:01:14 UTC: [5005ed5a458ecde1bac26b7ecc33fdd650a8b954] Deprecate package groups UI on content host detail page                                                                                                                                             
32661 - 2021-05-28 17:01:12 UTC: [c2ed31efd1cbd094177d5865d9a09934cf11301c] Old hostgroup with kickstart repository id aren't migrated to the new hostgroup content facet table.                                                                                                
32362 - 2021-06-01 18:02:27 UTC: [1619e1c5632e1810d06e48a79100dca7767b9c6f] katello pulp url setting in katello.yaml is no longer set by the installer and the default uses localhost rather than FQDN                                                                          
32320 - 2021-06-02 05:02:19 UTC: [d05262e60203bcb7f8e3577481c6449ee50ed5e2] "Current organization has no manifest imported" error when trying to import a manifest                                                                                                              
32582 - 2021-06-02 13:02:28 UTC: [f59a6cb35080bd735b795bd36daae094826ee654] Unable to delete Red Hat repository due to "bad" root repo data from older Katello                                                                                                                  
32396 - 2021-06-03 14:02:25 UTC: [69864057c2c55641dc1d9234fae323ee3d838eb8] Permissions for import/export                                                                                                                                                                       
32699 - 2021-06-03 20:02:32 UTC: [2a9c0187429680257a9e3e889aafa6294edfddc0] Katello 3.18 changed the way RH operating systems are named after sync                                                                                                                              
32624 - 2021-06-07 16:02:29 UTC: [8dfd072ef08a4dd9524dff998b305320bb83dc79] Client receives 403 forbidden when fetching RHEL content when using custom certificates                                                                                                             
32761 - 2021-06-09 16:08:16 UTC: [366df38d1e9051c5f4bc98a4460c2201d702a22f] Deb repo calls pulp2 when checking published during matching content check                                                                                                                          
32740 - 2021-06-09 17:08:13 UTC: [4bdc5a49b3e518bef69d28f2b99eed99c283122e] Katello:delete_orphaned_content still checking for pulp2 services and failing with 404s.                                                                                                            
32613 - 2021-06-10 18:08:10 UTC: [b9ac5ee55c6117fc70452a747c5e2b92e91ffe42] Fix pulpcore smart proxy error message                                                                                                                                                              
32784 - 2021-06-11 17:08:16 UTC: [c388df7f184859256880b02af99704667f78aefe] Error: undefined methodpulp_href’ for nil:NilClass`  when syncing capsule  
```

That's 23 items and the dates on them make sense based on when you did RC2.1. Your katello PR has 17 commits + 14 ignores here = 31 in total. 23 != 31 which means this list _and_ the items picked into Katello itself could be inaccurate. Can you recreate your tool_belt environment and double-check everything? Or, let me know if I am missing something :)

Reply:
I re-ran it and got 25.  The two in addition to the above are
```
32591 - 2021-05-17 20:16:08 UTC: [7c9aa78fb5c5c14ea15af759a9d36fd4a7368377] katello-agent uses incorrect ping mechanism
31197 - 2021-06-14 19:08:16 UTC: [427b67ebe2822d8f969fe7d5bc64b39af37c2b2a] upgrade pulp ansible plugin to 0.5.0
```

Reply:
Now that the cherry-pick situation is figured out in your Katello PR, are any of these 'empty' ignores necessary?

Reply:
@jturel removed :)